group_vars/location_manstein10/networks.yml

```diff
@@ -53,3 +53,8 @@ networks:
       # openwrt clients
       manstein10-m2-s: 7
       manstein10-m2-w: 8
+
+# South & West share a pole, North & East share another pole
+location__channel_assignments_11g_standard__to_merge:
+  mesh_2ghz_s: 1-20-10
```

Comment:
Falters default is channel 13.

its not possible to mesh using different channels

Reply:
So should the channel assignment be removed?

Reply:
Furthermore the key in that dict has to match the hostname

Reply:
> Okay. I can do it. But I would like to delete the existing stuff. But I cant since I have no control of the existing allocations.

Request help from someone with NIPAP access

Reply:
Just removed the commit with this chunk.

Reply:
removing would be the easiest for now, yes.